rcp/org.testeditor.rcp4.views.tcltestrun.tests/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilderTest.xtend

```diff
@@ -0,0 +1,209 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.rcp4.views.tcltestrun.model
+
+import org.testeditor.dsl.common.testing.AbstractTest
+import org.junit.Before
+import org.junit.Test
+
+class TestLogGroupBuilderTest extends AbstractTest {
+
+	TestLogGroupBuilder logGroupBuilder
+
+	@Before
+	def void setup() {
+		logGroupBuilder = new TestLogGroupBuilder
```

Comment:
In this PR I saw three different ways of creating the class under test.

1. `@InjectMocks MyClass instance`
2. `MyClass instance = new MyClass`
3. `new MyClass` in the setup block

I think we should always use the first one and do this consistently.